smolyay/smolyak.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+Created on Tue Jul  5 08:00:15 2022
+
+@author: mzf0069
+"""
+
```

Comment:
Do not include these in your files. The top part is not necessary, and we will write docstrings for the modules.

Reply:
Done.